packages/ptokens-assets-evm/src/ptokens-evm-provider.ts

```diff
@@ -18,9 +18,11 @@ export type MakeContractSendOptions = {
   /** The contract address. */
   contractAddress: string
   /** The value being sent with the transaction. */
-  value: string
+  value: bigint
```

Comment:
Isn't viem accepting strings for number? the string permits compatibility with different bignumber impementations